mlir/lib/Dialect/MIOpen/Generator/Conv2dGenerator.cpp

```diff
@@ -590,3 +590,71 @@ LogicalResult Conv2dGenerator::genConvModule(ModuleOp &module,
 
   return success();
 }
+
+int64_t Conv2dGenerator::getBwdDataNumberOfGemm() const {
```

Comment:
We already have `getBwdDataKernelCount()`, what's the difference between the two?

Reply:
getBwdDataKernelCount() returns the number of kernels with gemmK>0. 
I think I can call getBwdDataNumberOfGemm() inside getBwdDataKernelCount().